tts/global.def.lua

```diff
@@ -42,3 +42,34 @@ function printToAll(message, color) end
 ---@return string
 function stringColorToRGB(player_color) end
 
+
+---@shape tts__SnapPoint
+---@field position tts__Vector Position of the snap point. The position is relative to the entity's center (a local position).
+---@field rotation tts__Vector Rotation of the snap point. The rotation is relative to the entity's rotation (a local rotation).
+---@field rotation_snap boolean If the snap point is a "rotation" snap point.
+
+---@overload fun(parameters: table):string
+---@param parameters tts__SnapPoint[]
+---@return string
+function setSnapPoints(parameters) end
+
+--- Returns a table of sub-tables, each sub-table representing one snap point.
+---@return tts__SnapPoint[]
+function getSnapPoints() return {} end
+
+
+---@shape tts__VectorLineGroup
+---@field points tts__Vector[] An array of 2 or more points.  Lines are drawn between each consecutive pair of points.
```

Comment:
As per the above, could you please add a `@` prefix to the comment?

There's also some errant double spaces in the comment and also for `rotation` (2 lines below). That's minor enough that I'd normally let it slide, but seems as I'm requesting other changes, may as well throw it in.

Reply:
Fixed.  I thought I was reading "@Default" not "@comment" :)